columnphysics/icepack_shortwave.F90

```diff
@@ -1794,11 +1794,13 @@ subroutine compute_dEdd_3bd(                           &
          fswint, & ! SW interior absorption (below surface, above ocean,W m-2)
          fswthru   ! SW through snow/bare ice/ponded ice into ocean (W m-2)
 
-      real (kind=dbl_kind), intent(out) :: &
+      real (kind=dbl_kind), intent(inout) :: &
          fswthru_vdr, & ! vis dir SW through snow/bare ice/ponded ice into ocean (W m-2)
          fswthru_vdf, & ! vis dif SW through snow/bare ice/ponded ice into ocean (W m-2)
          fswthru_idr, & ! nir dir SW through snow/bare ice/ponded ice into ocean (W m-2)
-         fswthru_idf, & ! nir dif SW through snow/bare ice/ponded ice into ocean (W m-2)
+         fswthru_idf    ! nir dif SW through snow/bare ice/ponded ice into ocean (W m-2)
+
+      real (kind=dbl_kind), intent(inout) :: &
```

Comment:
Suggest recombining the two "real" sections to one that has
`real (kind=dbl_kind), intent(inout) :: &`
I think this just means changing line 1797 and leaving the rest as it was.

Reply:
Yes, changed!